src/hotspot/share/gc/parallel/psParallelCompact.hpp

```diff
@@ -961,6 +961,25 @@ inline void ParMarkBitMapClosure::decrement_words_remaining(size_t words) {
 // regions and regions compacting into themselves.  There is always at least 1
 // region that can be put on the ready list.  The regions are atomically added
 // and removed from the ready list.
+//
```

Comment:
This commentactually refers to the sentence: "// An alternate strategy is being investigated for this deferral of updating." above.

This sentence should be removed as this offers no information about the current algorithm. Fwiw I do not know that actually anyone is working on that, and if so, a CR in JIRA would be more appropriate.

Reply:
Removed!